src/services/layer-metadata/remote-metadata.helper.ts

```diff
@@ -0,0 +1,28 @@
+import { REMOTE_SERVICE_TYPE } from '../remote-layers/remote-layers.model'
+import { WmsHelper } from './wms.helper'
+import { WmtsHelper } from './wmts.helper'
+
+export class RemoteMetadataHelper {
+  private wmsHelper: WmsHelper
+  private wmtsHelper: WmtsHelper
```

Comment:
is there a reason for instantiating the helpers here? and store them as properties in the current helper?
Maybe I am missing something but you could simply return wmsHelper.getMetadata or wmtsHelper.getMetadata directly.

Reply:
Of course, thanks for paying attention to this, as I did not :-)